tests/test_calibrateImage.py

```diff
@@ -228,6 +236,18 @@ def test_run_no_optionals(self):
         self.assertNotIn("astrometry_matches", result.getDict())
         self.assertNotIn("photometry_matches", result.getDict())
 
+    def test_run_no_calibrate_pixels(self):
+        """Test that run() returns reasonable values to be butler put when
+        do_calibrate_pixels=False.
```

Comment:
I guess this came from my original wording, but this is much clearer.

```suggestion
        """Test that run() returns reasonable values when
        ``do_calibrate_pixels=False``.
```